addons/spreadsheet_account/tests/test_company_fiscal_year.py

```diff
@@ -44,6 +44,8 @@ def test_fiscal_year_reference_first_day(self):
         )
 
     def test_fiscal_year_with_company_id(self):
+        self.env.company.fiscalyear_last_day = 7
```

Comment:
if I understand correclty, the indeterministic behaviour came from some other test that would alter the fiscal year being run before ours. Shouldn't we make this test more reliant by fetching the current fiscal year info and use it to forge the domain in the assert equal ?

Reply:
>  from some other test

not from other tests, it's forced on the company for real
https://github.com/odoo/odoo/commit/65dacfc17bec5feba1216354c71ca542c1b762b2#diff-af9a7801f128865c705b82cc4001e7a142a681fb7548e458dccca4d7b5f6e320R12

> fetching the current fiscal year info and use it to forge the domain in the assert equal ?

I think it's easier to force a fiscal year rather than have dynamic/forged values in the assert